test-versions.txt

```diff
@@ -0,0 +1,2 @@
+#well known working crio tag/commit/branch
+crio_version=a95bbe9608d3b09b50651c82867bf7cfd83833e8
```

Comment:
Actually, this more recent commit `065960386fce7088bab3fc22af44ebbbf75d3641` is working.